.pre-commit-config.yaml

```diff
@@ -46,5 +46,5 @@ repos:
     rev: v0.13.0
     hooks:
       - id: detect-secrets
-        args: ['--baseline', '.secrets.baseline']
+        args: ["--baseline", ".secrets.baseline"]
```

Comment:
to fix the linting failure change
`repo: git@github.com:Yelp/detect-secrets` to `repo: https://github.com/Yelp/detect-secrets`